src/styles/utils/variables.scss

```
@@ -0,0 +1,12 @@
+$bodyFont: Roboto, sans-serif;
+$defFont-W: 500;
+$defFont-S: 12px;
+$logoSize: 40px;
+$textColorMain: #060b35;
+$textColorSecondary: #616070;
+$navLinkColor: #060b35;
```

No need to duplicate variables with the same values
```suggestion
$textColorMain: #060b35;
$textColorSecondary: #616070;
```